lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -971,6 +975,8 @@ def configure_disks(self, vm_obj):
             if 'type' in expected_disk_spec:
                 if expected_disk_spec.get('type', '').lower() == 'thin':
                     diskspec.device.backing.thinProvisioned = True
+                if expected_disk_spec.get('type', '').lower() == 'eagerzeroedthick':
```

Comment:
But I am wondering if these options are supposed to be mutual-exclusive, or rather that these are flags tou can enable/disable (it looks more like that).